### PR TITLE
Docs/orion ashley

### PR DESCRIPTION
**Pull request by ahl389**
Updated the landing page of the orion docs to better organize them

### CODE REVIEW

File: docs/source/workflows/predictors.rst [hunk 1]
```diff
@@ -125,4 +125,9 @@ The following example demonstrates how to use the python SDK to create a :class:
...
 
    # register predictor
-   predictor = project.predictors.register(shear_modulus_predictor)
\ No newline at end of file
+   predictor = project.predictors.register(shear_modulus_predictor)
+
+Predictor Reports
+--------------------
+
+A :doc:`predictor report <predictor_reports>` contains information about a machine-learned model and various performance metrics.
```
Comment (sparadiso) [inline docs/source/workflows/predictors.rst]: A predictor report _describes_ the model. What were its settings, what features are important to the model, etc. The report specifically does _not_ provide performance metrics. For that, you want to point to PerformanceWorkflows.

Comment (ahl389) [reply docs/source/workflows/predictors.rst]: Good to know - I actually just copied that from the first line of the existing predictor report doc. I will make an update in both locations.

File: docs/source/workflows/index.rst [hunk 2]
```diff
@@ -1,18 +1,50 @@
...
+Currently, there are two types of workflows on the AI Engine: the :doc:`DesignWorkflow <design_workflows>` and the :doc:`PerformanceWorkflow <performance_workflows>`.
+Workflows employ reusable modules in order to execute. 
+There are three different types of modules, and these are discussed in greater detail below.
+
+DesignWorkflow
+**********
+
+The :doc:`DesignWorkflow <design_workflows>` is the core AI workflow on the platform.
+This workflow generates scored candidates for sequential learning.
+It requires one of each of the modules listed above and executes in the following manner:
```
Comment (maxhutch) [inline docs/source/workflows/index.rst]:
```suggestion
It requires one of each of the modules listed below and executes in the following manner:
```

File: docs/source/workflows/index.rst [hunk 3]
```diff
@@ -1,18 +1,50 @@
...
+
+After a given number of iterations, candidates are ranked according to their score and the best materials are returned.
+(Here, the best materials are those that are most likely to optimize an objective and satisfy a set of constraints.)
+
+Design workflows are further parameterized by :doc:`Scores <scores>`, which codify experimental objectives and constraints on desired candidates, and define the strategy for candidate acquisition.
+
+PerformanceWorkflow
+**********
+
+The :doc:`PerformanceWorkflow <performance_workflows>` is used to perform analysis on a predictor module. The PerformanceWorkflow exists to help the user understand how well their predictor module works with their data: in essence, it describes the trustworthiness of their model. These outcomes are captured in a series of performance metrics.
```
Comment (maxhutch) [inline docs/source/workflows/index.rst]:
```suggestion
The :doc:`PerformanceWorkflow <performance_workflows>` is used to perform analysis on a predictor module.
The PerformanceWorkflow exists to help the user understand how well their predictor module works with their data: in essence, it describes the trustworthiness of their model.
These outcomes are captured in a series of performance metrics.
```
(No more than) one sentence per line.  This makes diffs easier to track.

File: docs/source/workflows/predictors.rst (hunk 1 above)
Comment (ahl389) [reply docs/source/workflows/predictors.rst]: @sparadiso tried to clarify this, please review

**Pull request merged**